src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,50 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer(Long id, String name, String country) {
+        this.id = id;
+        this.name = name;
+        this.country = country;
+    }
```

Comment:
```suggestion
    public Manufacturer( String name, String country) {
        this.name = name;
        this.country = country;
    }
```

I think we can path just name and country here and not not pass null`s in main method